### PR TITLE
Couple of utility functions + fix for #17

### PR DESCRIPTION
**Pull request by SkepticMystic**
- [x] I agree to the contribution guidelines laid out [here](https://github.com/obsidian-community/obsidian-community-lib#contributing) in the readme.

### CODE REVIEW

File: dist/interfaces.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
```
Comment (phibr0) [inline dist/interfaces.js:2]: This looks odd, I mean there are no interfaces in JS and I guess its fine, but this is just an empty file

**Comment by SkepticMystic** [comment]
You'll see I bumped to `1.0.0` as this had a breaking change. I renamed two things, so I think that means users will have to change the name if they use those features

**Review by phibr0 — APPROVED** [review]

File: dist/interfaces.js (hunk 1 above)
Comment (SkepticMystic) [reply dist/interfaces.js:2]: Huh, that is strange. I'm not sure what's happening here, but things still work

Comment (phibr0) [reply dist/interfaces.js:2]: This is the file that only holds the interface, interfaces don't exist in JS (only TS) which is why the transpiled file is basically empty

Comment (SkepticMystic) [reply dist/interfaces.js:2]: Oh right, so it shouldn't add it in the first place

**Pull request merged**